src/mbgl/map/transform_state.cpp

```diff
@@ -64,6 +64,14 @@ void TransformState::getProjMatrix(mat4& projMatrix) const {
 
     matrix::translate(projMatrix, projMatrix, pixel_x() - size.width / 2.0f,
                       pixel_y() - size.height / 2.0f, 0);
+
+    const auto circumferenceOfEarth = 2 * M_PI * 6378137;
```

Comment:
[`EARTH_RADIUS_M`](https://github.com/mapbox/mapbox-gl-native/blob/6f708ac5458fe332e25ca398431928b7ff5ba404/include/mbgl/util/constants.hpp#L31) is available.